### DISCUSSION TITLE
Cannot find throttleconfiguration.ini

### DISCUSSION THREAD

**Discussion by Mike777-200ER** [flybywiresim/aircraft#2333] [Help]
I have a non-standard installation - Windows is on C: but MSFS is in D:\FS2020 Program . I have looked everywhere, done searches, but no luck.  Is there a file within the A320NX folder (in Community) that points to the ThrottleConfiguration.ini file?  My Community folder is at top level of D:\FS2020 Program folder.

Thanks

**Comment by Benjozork** [comment]
We don't provide technical support for the custom fly-by-wire system version you are referring to.

**Comment by Nadelbaum** [comment]
While the dev team doesn't provide support for unreleased stuff, which is understandable, maybe this will help you out:

https://github.com/flybywiresim/a32nx/tree/fbw/docs

**Reply by Mike777-200ER** [reply]
Thanks Nadelbaum.  I already had read that document but the Throttleconfiguration.ini is not where that doc says - probably because FS isn't installed in the standard location.  I'm thinking that the fbw version isn't installing it due to my non-standard installation, as I've looked in a lot of locations and in fact searched both entire drives.

**Reply by Mike777-200ER** [reply]
Throttleconfiguration.ini has now turned up on my computer and is working, after downloading the latest version of A32NX-master-cfbw.zip today (12-12-20)  

I have a non-standard installation where MSFS is on  D Drive but Windows is on C.

D:\WpSystem\[A BUNCH OF NUMBERS]\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\a32nx\work

Mike
